packages/editor/src/components/post-featured-image/style.scss

```
@@ -66,6 +66,11 @@
 
 .editor-post-featured-image__toggle {
 	box-shadow: inset 0 0 0 $border-width $gray-400;
+
+	&:focus:not(:disabled) {
+		// Allow smooth transition between focused and unfocused box-shadow states.
+		box-shadow: 0 0 0 currentColor inset, 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color, #3858e9);
```

```suggestion
		box-shadow: 0 0 0 currentColor inset, 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
```

We won't need fallbacks in this package 👍